### ISSUE TITLE
jtframe mra should be able to parse all cores

### ISSUE THREAD

**Issue by jotego** [jotego/jtframe#74]
At the moment `jtframe mra` can only parse one core at a time. It would be better if it could parse all of them in a single call.

Current behavior, for example in the JTKICKER repository
```
cd $ROM
jtframe mra kicker
jtframe mra yiear
# etc.
```
Desired behavior:
```
jtframe mra --all
```

The current structures are designed to parse mame.xml looking for a single driver at a time. Although many _machine_ objects can get extracted, they are always parsed against a single configuration file (a single _mame2mra.toml_ file)

The easiest thing is to add the multifile capability in the _jtframe mra_ command [itself](https://github.com/jotego/jtframe/blob/9e4bbeb471b933a607ff419ff881a25581d6a59d/src/cmd/mra.go). 
The best solution is to adapt the code to be able to parse multiple configuration files so mame.xml is parsed a single time.

- [ ] Make jtframe operate always on the $ROM folder
- [ ] Support to parse multiple confiuration files (each one located in its own core folder **cfg/mame2mra.toml**)
- [ ] Modify the `jtframe mra` command to support the new `--all` argument

**Comment by Flor2259** [comment]
let me give it a go

**Comment by jotego** [comment]
This was included in ca6bab7ea as part of jtupdate
